cmd/servicelog/post.go

```diff
@@ -146,54 +154,69 @@ func (o *PostCmdOptions) Run() error {
 		}
 	}()
 
-	// Retrieve matching clusters
+	// Merge OCM filters from all custom filter-related flags
 	if o.filtersFromFile != "" {
-		if len(filterParams) != 0 {
+		if len(o.filterParams) != 0 {
 			log.Warnf("Search queries were passed using both the '-q' and '-f' flags. This will apply logical AND between the queries, potentially resulting in no matches")
 		}
 		filters := strings.Join(strings.Split(strings.TrimSpace(o.filtersFromFile), "\n"), " ")
-		filterParams = append(filterParams, filters)
+		o.filterParams = append(o.filterParams, filters)
 	}
+
+	// Combine existing OCM filters with any cluster id-related flags
 	var queries []string
 	if o.clustersFile != "" {
 		contents, err := o.accessFile(o.clustersFile)
 		if err != nil {
-			log.Fatalf("Cannot read file %s: %q", o.clustersFile, err)
+			return fmt.Errorf("cannot read file %s: %v", o.clustersFile, err)
```

Comment:
Any specific reason to not wrap the error here?
```suggestion
			return fmt.Errorf("cannot read file %s: %w", o.clustersFile, err)
```

Reply:
Ah, just that we don't normally unwrap errors, but no real reason not to prevent it :D